src/utils/date.ts

```diff
@@ -1,6 +1,9 @@
 import { DateTime } from 'luxon';
 
-const formatDate = (date: string | DateTime) => {
+export const parseDate = (date: string) =>
```

Comment:
I'm trying to think of a clearer name for what this does. A bit wordy, but maybe `convertIsoToUtcDateTime`